package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "docs-template",
```

Comment:
Multiple problems with this package file - 
i) It is still named docs-template rather than after this module
ii) Description is for the template
iii) Version should be a proper version - presumably matching that of the version of the first discovery tool that it is the docs for
iv) no licence field
v) no repository field
vi) On our LTS npm (2.14.8), npm install produces the following warnings:

npm WARN deprecated URIjs@1.16.1: package renamed to "urijs" (lower-case), please update accordingly
docs-core@0.0.2-SNAPSHOT node_modules\docs-core
└── URIjs@1.16.1